components/Layout/Nav.tsx

```diff
@@ -82,37 +82,52 @@ const Nav = () => {
-              <p className={'text-[#B999FA] text-[16px]'}> MINE.FM Live Broadcasts </p>
-            </div>
-            <div className={'animate-marquee2 absolute top-0 flex items-center whitespace-nowrap'}>
-              <div className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'} />
-              <p className={'text-[#B999FA] text-[16px]'}> MINE.FM Live Broadcasts </p>
-            </div>
+      {isMobile ? (
+        <div
+          className={
+            'flex w-fit h-12 border-solid border-[#654A8F] rounded-md overflow-x-hidden overflow-y-hidden relative cursor-pointer md:ml-12'
+          }
+        >
+          <div
+            className={
+              'animate-marquee flex items-center justify-center pt-1 whitespace-nowrap'
+            }
+          >
+            <div
+              className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'}
+            />
+            <p className={'text-[#B999FA] text-[16px]'}> MINE.FM Live Broadcasts </p>
           </div>
-        ) : (
-          <div className={'flex items-center justify-center w-2/5 h-12 border-solid border-[#654A8F] rounded-md overflow-x-hidden overflow-y-hidden relative cursor-pointer md:ml-16'}>
-            <div className={'flex items-center justify-center pt-1 whitespace-nowrap'}>
-              <div className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'} />
-              <p className={'text-[#B999FA] text-center text-[16px]'}> MINE.FM Live Broadcasts </p>
-            </div>
-
+          <div
+            className={
+              'animate-marquee2 absolute top-0 flex items-center whitespace-nowrap'
+            }
+          >
+            <div
+              className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'}
+            />
+            <p className={'text-[#B999FA] text-[16px]'}> MINE.FM Live Broadcasts </p>
           </div>
-        )
-      }
-
-      <div className={navActions}>
-        <Link
-          key={'about-us'}
-          href={'/about-us'}
+        </div>
+      ) : (
+        <div
+          className={
+            'flex items-center justify-center w-2/5 h-12 border-solid border-[#654A8F] rounded-md overflow-x-hidden overflow-y-hidden relative cursor-pointer md:ml-16'
+          }
         >
-        <p className={`m-2 text-xl text-[#B999FA] cursor-pointer hover:text-purple-800 ${isMobile ? 'hidden' : null}`}> ABOUT US </p>
+          <div className={'flex items-center justify-center pt-1 whitespace-nowrap'}>
+            <div
+              className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'}
+            />
+            <p className={'text-[#B999FA] text-center text-[16px]'}>
+              {' '}
+              MINE.FM Live Broadcasts{' '}
+            </p>
+          </div>
+        </div>
+      )}
 
-        </Link>
+      <div className={navActions}>
         {address && <NavMenu hasAccount={hasAccount} signerAddress={address} />}
```

Comment:
we should add the ABOUT US link to the nav menu drop down when users are connected as well.

Reply:
added it while users are connected. There isn't much of a design for where the about us dropdown should position the text link and I think it looks a little weird for logged in users. Do you have any recs for that? @blobitty

Reply:
my b put comment in the wrong spot: https://github.com/Brick-Creative-Studio/mine-fm/pull/67#discussion_r1434594338